SyslogNet.Client/Transport/ISyslogMessageSender.cs

```diff
@@ -9,5 +10,8 @@ public interface ISyslogMessageSender : IDisposable
 		void Reconnect();
 		void Send(SyslogMessage message, ISyslogMessageSerializer serializer);
 		void Send(IEnumerable<SyslogMessage> messages, ISyslogMessageSerializer serializer);

```

Comment:
If the sync interface implementation are just going to be calling .Wait() on the Async then should they be exposed? If this is called from a UI thread then this will deadlock and the consumer can't fix it.